proxy/http/HttpTransact.cc

```diff
@@ -5522,7 +5527,10 @@ HttpTransact::initialize_state_variables_from_request(State* s, HTTPHdr* obsolet
 
   if (!s->server_info.name || s->redirect_info.redirect_in_process) {
     s->server_info.name = s->arena.str_store(host_name, host_len);
-    s->server_info.port = incoming_request->port_get();
+    ats_ip_copy(&s->server_info.dst_addr, &s->request_data.src_ip); //initialize w/ junk
+    s->server_info.dst_addr.port() = incoming_request->port_get();
+    ip_port_text_buffer ip_s;
+    DebugTxn("http_trans", "init_from_request server: %s port: %d incoming port: %d", ats_ip_nptop(&s->server_info.dst_addr.sa, ip_s, sizeof(ip_s)), s->server_info.dst_addr.port(), incoming_request->port_get());
```

Comment:
i found these additional messages in the debug logs to be very helpful but can drop them if others want them gone.